applications/plugins/SofaPython/Binding_BaseContext.cpp

```diff
@@ -106,7 +121,12 @@ extern "C" PyObject * BaseContext_createObject_Impl(PyObject * self, PyObject *
             {
                 if (PyString_Check(value))
                     desc.setAttribute(PyString_AsString(key),PyString_AsString(value));
-                else
+                else if (PyList_Check(value)){
+                    SP_MESSAGE_WARNING( "Sofa.Node.createObject("<<type<<"), automatic conversion from List for attribute "<<PyString_AsString(key)<<")..." )
```

Comment:
ok, now I understand it, I like it ;)
I would even print no warning at all in that situation, but I would print one in the `else`/default case!